src/store/modules/images.js

```diff
@@ -259,6 +266,42 @@ const actions = {
      *     taken the active user or not.
      */
 
+  // we want to group images as they load and we group them based on their SMARTSTK value
+  // we dispatch this action in multiple places
+  group_images ({ commit, state, rootState, dispatch }) {
+    const currentSite = rootState.site_config.selected_site
+    let grouping_images = JSON.parse(JSON.stringify(state.grouped_images))
+    const recent_images = state.recent_images
+    if (grouping_images.site && grouping_images.site !== currentSite) {
+      dispatch('reset_grouped_images')
+      grouping_images = { site: currentSite }
+    } else if (!grouping_images.site) {
+      grouping_images.site = currentSite
+    } else if (grouping_images.site && grouping_images.site == currentSite) {
+      console.log('continuing')
+    }
+    for (let i = 0; i < recent_images.length; i++) {
+      const img = recent_images[i]
+      const header = img && img.header
+      let SMARTSTK = header && header.SMARTSTK
+      if (SMARTSTK === 'no') {
```

Comment:
Maybe add a comment about what we're doing when the SMARTSTK header keyword is 'no'

Reply:
Done!